### DISCUSSION TITLE
Dealing with post-processing

### DISCUSSION THREAD

**Discussion by milenazuccarelli** [google/model-viewer#2201] [Q&A]
Not a bug but I see there are some open tickets regarding the matter - just wondering in the meantime what the best approach for adding post-processing effects would be (film grain and bloom). 

Do you have any suggestions or guidance for this? Thanks!

**Comment by elalish** [comment]
We don't have any plans to work on that currently. If Three.js has post processing that would work for you, your best bet would probably be to fork model-viewer and add those effects to our existing three.js renderer. If you think up a general API for post-processing effects, we'd welcome a contribution.
